fuel-core-interfaces/src/block_importer.rs

```diff
@@ -3,7 +3,7 @@ use super::model::{
     FuelBlock,
     SealedFuelBlock,
 };
-use fuel_types::Bytes32;
+use fuel_vm::fuel_types::Bytes32;
```

Comment:
It may look cleaner to use the common re-export instead of importing fuel-vm directly in every circumstance.

```suggestion
use crate::common::fuel_types::Bytes32;
```